Assets/Scripts/SaveLoad/DataModels/OnePlayerData.cs

```diff
@@ -0,0 +1,39 @@
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+[System.Serializable]
+public class OnePlayerData
+{
+	public string PlayerName {get; set;}
```

Comment:
Im not sure if this is some kind of convention but this spacing looks off to me

Reply:
sorry that's my VS Code harassing me. I don't know why it stopped detecting tabs on save